dbt/adapters/impala/connections.py

```diff
@@ -265,10 +285,25 @@ def add_query(
             configuration = {"paramstyle": "format"}
             cursor.execute(sql, bindings, configuration)
 
+            elapsed_time = time.time() - pre
+
+            query_status = str(self.get_response(cursor))
+
+            payload = {
+                "event_type": "dbt_impala_end_query",
+                "sql": log_sql,
+                "elapsed_time": "{:.2f}".format(elapsed_time),
+                "status": query_status,
+                "connection_name": connection.name,
```

Comment:
Looks like result of connection_name and node_id(model name) will be same. We can then remove of those.

Reply:
done, removed connection_name